.github/workflows/build-lint-test.yml

```diff
@@ -22,7 +22,7 @@ jobs:
 
```

Comment:
it seems you have DevOps / GHA / local development related changes.

feel free to create a separate PR for those and get that merged right away.

Reply:
I think we're close to merging this as is